src/main/java/org/apache/datasketches/Job.java

```diff
@@ -22,7 +22,7 @@
 import static org.apache.datasketches.Files.isFileValid;
 import static org.apache.datasketches.Files.openPrintWriter;
 import static org.apache.datasketches.Util.milliSecToString;
-import static org.apache.datasketches.memory.Util.getResourcePath;
+import static org.apache.datasketches.Util.getResourcePath;
```

Comment:
getResourcePath isn't under memory in branch 1.1.X as the pom.xml requires

Reply:
looks good

Reply:
This is my fault.  

datasketches-java 1.1.0 has the following three methods in o.a.datasketches.Util:
```
public static String getResourcePath(final String shortFileName)
public static File getResourceFile(final String shortFileName) 
public static byte[] getResourceBytes(final String shortFileName)
```
These methods were improved after a subtle bug was discovered by one of our mentors where the tests were failing on his laptop where he had installed the release in a directory that contained a space character in the path name.  Actually, according to the Java spec, this is illegal, nevertheless these methods now contain a fix to work around this issue.  This fix is in datasketches-java 1.1.0, which was released after datasketches-memory 1.1.0 was released.  I recognized that some of the memory could would have this same problem so I added these methods to the memory.Util as well.  But they don't appear as a release until memory-1.2.0.  

So the real fix for this is to change the memory dependency to 1.2.0-incubating.  

Because I tend to run the characterization tests linking to memory/master on my system, I didn't recognize that this was a problem.  I will submit a PR to fix this.